docs/example.ts

```diff
@@ -72,7 +75,7 @@ const ctype = new Kilt.CType(rawCtype)
 
 // Store ctype on blockchain
 // ! This costs tokens !
-// Also note, that the completly same ctype can only be stored once on the blockchain.
+// Also note, that the completely same ctype can only be stored once on the blockchain.
```

Comment:
I don't like completely here anyway. I would remove it, or change it. 

Suggestion: 

Also note, that the absolutely identical ctype can only be stored once on the blockchain.